waiter/src/waiter/state.clj

```diff
@@ -31,10 +31,17 @@
 
 ;; Router state maintainers
 
-(defn- healthy? [status-tags]
+(defn- healthy?
```

Comment:
This and `expired?` might make more sense accepting an instance's state instead of the `status-tags` map directly, since they tell you whether an instance is healthy, not whether a set of status tags is healthy.  Also it may improve readability where they are used.